tgui/packages/tgui-panel/settings/reducer.js

```diff
@@ -26,6 +26,8 @@ const initialState = {
   lineHeight: 1.2,
   theme: 'dark',
   adminMusicVolume: 0.5,
+  MessageStackInSeconds: 30,
+  MaxTotalMessage: 10000,
```

Comment:
```suggestion
  messageStackInSeconds: 30,
  maxTotalMessage: 10000,
```

js case styling is camelCase for identifier names (this will need to change in refs too)

Reply:
addressed
god i hate js